src/radical/pilot/configs/agent_default.json

```diff
@@ -36,19 +36,19 @@
     #
     "bridges" : {
         "agent_staging_input_queue"  : { "kind"      : "queue",
-                                         "log_level" : "error",
+                                         "log_level" : "debug",
```

Comment:
Should 'debug' be the default level in our releases? I would assume it should be 'error'?

Reply:
Right, will revert to `error`.